### PR TITLE
Add Thinknear to list of users

### PR DESCRIPTION
**Pull request by d3cay1-zz**
Make sure you have checked _all_ steps below.

### JIRA
- [x] My PR addresses the following [Airflow JIRA](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "[AIRFLOW-XXX] My Airflow PR"
    - https://issues.apache.org/jira/browse/AIRFLOW-2251

### Description
- [x] Here are some details about my PR, including screenshots of any UI changes:
Adding Thinknear to README as we've been a user of Airflow for over a year now in production

### Tests
- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:
Doc change, no test

### Commits
- [x] My commits all reference JIRA issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

- [ ] Passes `git diff upstream/master -u -- "*.py" | flake8 --diff`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -196,6 +196,7 @@ Currently **officially** using Airflow:
 1. [Zenly](https://zen.ly) [[@cerisier](https://github.com/cerisier) & [@jbdalido](https://github.com/jbdalido)]
 1. [Zymergen](https://www.zymergen.com/)
 1. [99](https://99taxis.com) [[@fbenevides](https://github.com/fbenevides), [@gustavoamigo](https://github.com/gustavoamigo) & [@mmmaia](https://github.com/mmmaia)]
+1. [Thinknear](https://www.thinknear.com/) [[@d3cay1](https://github.com/d3cay1), [@ccson](https://github.com/ccson), & [@ababian](https://github.com/ababian)]
```
Comment (jgao54) [inline README.md]: nit-picking: alphabetical order please :)

Comment (d3cay1-zz) [reply README.md]: Sure, I didn't notice the ordering.

**Comment by d3cay1-zz** [comment]
Is there a way to kick off the travis build again?  The error is definitely not related to this README change.  Looks to be a database cleanup problem.

**Comment by cjgu** [comment]
Closing and reopening the PR will trigger a rerun.

**Review by Fokko — APPROVED** [review]
@d3cay1 Thanks extending the list, merging since the error is unrelated.

**Pull request closed without merging**